_posts/2020-07-06-snowplow-android-tracker-1.5.0-released/index.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Snowplow Android Tracker 1.5.0 released"
+title-short: Snowplow Android Tracker 1.5.0
+tags: [snowplow, android, tracker]
+author: Alex Benini
+category: Releases
+permalink: /blog/2020/07/06/snowplow-android-tracker-1.5.0-released/
+discourse: false
+---
+
+We are pleased to announce [Snowplow Android Tracker](https://github.com/snowplow/snowplow-android-tracker) 1.5.0. [Version 1.5.0]((https://github.com/snowplow/snowplow-android-tracker/releases/tag/1.5.0)) introduces improvements to event management, avoiding duplication of the event ID when the same event object is reused multiple times. Additionally, it includes reporting of processing errors in background threads so that they can be tracked using the diagnostics tracking introduced in version 1.4.0 as well.
+
+Read on below for:
+
+1. [Avoid duplicate event IDs when reusing event objects](#duplicates)
+2. [Report errors in background threads](#diagnostic)
+3. [Updates and bug fixes](#updates)
+4. [Documentation](#documentation)
+5. [Getting help](#help)
+
+<!--more-->
+
+<h2 id="duplicates">1. Avoid duplicate event IDs when reusing event objects</h2>
+
+In previous versions, the event objects created by the user already contained the `event_id` and `dvce_created_tstamp`. If an event object is then tracked multiple times, i.e. 
+
+{% highlight java %}
+let e = SPScreenView.build {
+                $0.setName("screen1")
+                $0.setTransitionType("trans1")
+}
+tracker.track(e)
+tracker.track(e)
+{% endhighlight %}
+
+this will lead to multiple events with the same event ID and device created timestamp being sent to the collector [GitHub issue #390](https://github.com/snowplow/snowplow-android-tracker/issues/390). Version 1.5.0 moves the assignment of event ID and device created timestamp into the event processing, so that the event object no longer contains them. This means you can now track the same event object multiple times, and each event will have a unique event ID. 
+
+
+<h2 id="diagnostic">2. Report errors in background threads</h2>
+
+In version 1.4.0, `trackerDiagnostic` events where introduced. While the tracker always manages errors internally (avoiding crashes of the app and assuring that no events are lost), this new method enables the tracker to report errors to a Snowplow collector as `diagnostic_error` events. From 1.5.0, we now also report errors that happen in background threads [GitHub issue #394](https://github.com/snowplow/snowplow-android-tracker/issues/394). Therefore, the tracker can now report errors that occur during tracking and sending of events to the collector.
+
+As long as error tracking is enabled, the errors occuring in background threads will be sent: 
```

Comment:
> As long as **error tracking** is enabled

My concern is that it could be confused with the exception tracking, which tracks exceptions/error in the customer app. Do you think they could confuse it?
Can "As long as diagnostic tracking is enabled" be a better alternative in case?